src/doc/unstable-book/src/language-features/half-open-range-patterns-in-slices.md

```diff
@@ -0,0 +1,21 @@
+# `half_open_range_patterns_in_slices`
+
+The tracking issue for this feature is: [#67264]
+It is part of the `#![exclusive_range_pattern]` feature,
+tracked at [#37854].
+
+[#67264]: https://github.com/rust-lang/rust/issues/67264
+[#37854]: https://github.com/rust-lang/rust/issues/37854
+-----
+
+This feature allow using half open range patterns in slices.
```

Comment:
You should probably point out that you can use them today by just wrapping them in parenthesis. And also it's just top-level half-open range patterns.

Reply:
> You should probably point out that you can use them today by just wrapping them in parenthesis.

I don't understand. In the doc example that I used, how would this be wrapped?

> And also it's just top-level half-open range patterns.

How about ?
```suggestion
This feature allow using top-level half-open range patterns in slices.
```

Reply:
I can't actually test it, but I would think:

`let [(a @ 3..), (b @ ..3), (c @ 4..6), ..] = xs;`

Reply:
Done.